src/Store/TaggedStore.php

```diff
@@ -115,14 +115,19 @@ public function getMultiple(array $keys) : StoreResult
                 unset($r[$k]);
                 continue;
             }
-            $allTags = array_merge($allTags, array_keys($tv->tags));
+            $allTags[] = array_keys($tv->tags);
         }
+        $allTags = array_merge(...$allTags);
```

Comment:
Curious what the reason was here for those changes? Perf?

Reply:
Minor but yeah.